src/components/DropZone.tsx

```diff
@@ -1,9 +1,27 @@
 import React from "react";
 import Dropzone from "react-dropzone";
 
+const onFileDrop = (acceptedFile: any): void => {
+  const reader = new FileReader();
+  if (reader.error) {
```

Comment:
in which case could this happen ? you created the reader just above and nothing happened in between ?